libs/beacon/include/beacon/beacon_setup_service.hpp

```diff
@@ -19,14 +19,16 @@
 
 #include "core/service_ids.hpp"
 #include "core/state_machine.hpp"
+#include "dkg/dkg_complaints_manager.hpp"
```

Comment:
is this moving to beacon/ ?

Reply:
Wasn't sure about whether everything was going to be moved into the same directory so left it.